nuxt.config.ts

```diff
@@ -1,6 +1,22 @@
 import { defineNuxtConfig } from 'nuxt3'
 
-// https://v3.nuxtjs.org/docs/directory-structure/nuxt.config
+// // https://v3.nuxtjs.org/docs/directory-structure/nuxt.config
 export default defineNuxtConfig({
 
+//   buildModules: ['@nuxtjs/style-resources'],
+
+//   styleResources: {
+//    // your settings here
+//    scss: ['./styles/*.scss']
+//   }
+  vite: {
+    css: {
+        preprocessorOptions: {
+            scss: {
+                additionalData: '@import "@/styles/index.scss";',
+            },
+        },
+    },
+}
 })
```

Comment:
@mishschmid So one thing interesting though is how to configure `@nuxtjs/style-resources`. Found this solution here https://githubmemory.com/repo/nuxt-community/style-resources-module/issues/190 

Plus I had to downgrade `sass:1.32.13` to fix these kinda of warnings `DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.`